arch/x86/crc32_fold_pclmulqdq.c

```diff
@@ -5,6 +5,7 @@
  * A white paper describing this algorithm can be found at:
  * https://www.intel.com/content/dam/www/public/us/en/documents/white-papers/fast-crc-computation-generic-polynomials-pclmulqdq-paper.pdf
  *
+ * Copyright (c) 2016 Marian Beermann (support for initial value)
```

Comment:
Uppercase `(C)` to be similar to line below it? Also maybe include Intel Copyright first..